src/Extension/AssetStoreURLHandler.php

```diff
@@ -29,7 +29,7 @@ public function updateResponse(HTTPResponse &$response, string $asset, array $co
         }
 
         // We are unable to progress if there is no current Controller
-        if (!Controller::has_curr()) {
+        if (!(Controller::curr() instanceof Controller)) {
```

Comment:
```suggestion
        if (!Controller::curr()) {
```
It will always either be a controller or `null` - so you can do a falsy check or a `Controller::curr() !== null` check - but instanceof reads like there's some other types we'd expect it to return.

Reply:
Thank you for flagging that. I have updated this to `if (!Controller::curr()) {`